source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -165,81 +220,152 @@ Table 3.p (below): List of Data_Type Values
 
    * - Report
-     - A report.
+     - A document presenting information in an organized format for a specific audience and purpose, such as a policy report.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
+       Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, DR, TR, IR
+
+   * - Software
+     - Source code or compiled software, or a virtual notebook environment used for programming.
+     - Data_Repository\ |br|\ |lb|
        Repository
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1
+     - PR, IR
 
-   * - Repository_Item
-     - A generic classification used for items stored in a repository.
-     - Repository
-     - PR, IR\ |br|\ |lb|
-       PR_P1
+   * - Sound
+     - A form of multimedia, typically describing materials that are audio-only, such as radio programmes.
+     - Multimedia\ |br|\ |lb|
+       Multimedia_Collection
+     - PR, DR, IR
+
+   * - Standard
+     - A document outlining processes agreed and established by authority or by general consent (e.g. materials from NISO).
+     - A&I_Database\ |br|\ |lb|
+       Aggregated_Full_Content\ |br|\ |lb|
+       Discovery_Service\ |br|\ |lb|
+       Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, DR, TR, IR
 
    * - Thesis_or_Dissertation
-     - A thesis or dissertation.
+     - A thesis or dissertation, such as one written by a PhD candidate.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
-       Repository
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1
+       Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, DR, TR, IR
 
    * - Unspecified
-     - It is not possible to classify the content because there isn’t sufficient information available.
-
-       Note that report providers are expected to make all reasonable efforts to classify the content and that using Data_Type Unspecified may affect the result of an audit, see :numref:`missing-values` for details.
+     - Content that cannot be classified by any of the other Data_Types due to lack of sufficient information. Note that report providers are expected to make all reasonable efforts to classify the content. Using Unspecified will give rise to a Warning in the Validation Tool.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Data_Repository\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
        Repository\ |br|\ |lb|
        Scholarly_Collaboration_Network
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1
+     - PR, DR, TR, IR\
+
+\*Some Data_Types are associated with Parent_Data_Types. For example, Data_Type Article has Parent_Data_Type Journal, while Data_Type Book_Segment has Parent_Data_Type Book.
+
+* Host_Types that MUST offer an IR can choose to provide Parent_Data_Type and other relevant parent information if it is available.
```

Comment:
Shouldn't this be a MUST rule? I would find it strange to have a SHOULD rule for Host_Types that choose to offer the report while Host_Types that MUST offer the report can choose whether to include that information or not if it is available.